src/features/home/sections/QASection/components/Accordion.module.scss

```diff
@@ -0,0 +1,66 @@
+@import '@/theme/variables';
+@import '@/theme/mixins';
+
+.containerAccordion {
+  border-bottom: 1px solid #000000;
+  width: 43.125rem; // 690px / 16;
+  padding: 8px;
+  box-sizing: border-box;
+}
+
+.childrenAccordion {
+  @include text-main;
+  padding: 10px 0 10px 70px;
+  overflow: hidden;
+  height: 0;
+  opacity: 0;
+  transition:
+    height 0.6s ease-in-out,
+    opacity 0.6s ease-in-out;
+}
+
+.childrenOpen {
+  height: auto;
+  opacity: 1;
+}
+
+.childrenClosed {
+  height: 0;
+  opacity: 0;
+  padding: 0;
+}
+
+.headerAccordion {
```

Comment:
use nesting and you don't need unique names (.header instead .headerAccordion)

Reply:
Also I already made commit with this change before suggesting. My mistake. What is your opinion about applying or not padding 0?

Reply:
Sorry for confusion. It would be better to apply not padding 0, but just padding-top: 0; padding-bottom: 0. Or maybe there are another ways to fix that extra gap.

Reply:
Ok. Sorry, I removed padding by accident